### PR TITLE
Implement `create` command in `frontity`.

### PR DESCRIPTION
**Pull request by orballo**

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   ],
   "homepage": "https://frontity.org",
   "license": "Apache-2.0",
-  "main": "src/index.js",
+  "main": "dist/scripts/index.js",
```
Comment (luisherranz) [inline packages/core/package.json:10]: Don't you need to add a `"types"` field here?

File: packages/core/src/scripts/build.ts [hunk 2]
```diff
@@ -60,14 +60,12 @@ const build = async ({
   if (mode === "production") await remove(join(outDir, "bundling"));
 };
 
-(process as NodeJS.EventEmitter).on("unhandledRejection", (error: Error) => {
-  console.error(error);
-  process.exit(1);
-});
+// (process as NodeJS.EventEmitter).on("unhandledRejection", (error: Error) => {
```
Comment (luisherranz) [inline packages/core/src/scripts/build.ts]: You can remove all the comments of the `dev`, `build` and `serve` files to keep them clean.

**Review by luisherranz — CHANGES_REQUESTED** [review]
Apart from those two comments I think it's great!

**Review by DAreRodz — APPROVED** [review]
👌

**Review by luisherranz — APPROVED** [review]
👏

**Pull request merged**